package.json

```diff
@@ -79,6 +79,7 @@
 		},
 		"patchedDependencies": {
 			"@cloudflare/component-listbox@1.10.6": "patches/@cloudflare__component-listbox@1.10.6.patch",
+			"debug@4.4.1": "patches/debug@4.4.1.patch",
```

Comment:
I guess there is no way to move that to the preset?

Reply:
Unfortunately, no. I did a bit of search on pnpm and it seems to be by design because pnpm uses symlinks and there are only one copy of the package.